auto_round/autoround.py

```diff
@@ -1394,7 +1403,9 @@ def quantize_via_rtn_blockwise(self, all_to_quantized_module_names: list[str]) -
                 if self.device_map is not None:
                     accelerate.hooks.remove_hook_from_submodules(block)
 
-                if is_nv_fp(self.act_data_type) and any("nv_fp" in format_ for format_ in self.formats):
+                if (is_nv_fp(self.act_data_type) and any("nv_fp" in format_ for format_ in self.formats)) or (
+                    any("auto_round:fp8" in format_ for format_ in self.formats)
```

Comment:
Would it be possible to refactor this check into a function to improve readability? Thanks